app/src/main/java/org/jellyfin/mobile/MainActivity.kt

```diff
@@ -70,15 +68,17 @@ class MainActivity : AppCompatActivity(), WebViewController {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_webapp)
+        setContentView(R.layout.activity_main)
 
         // Bind player service
         bindService(Intent(this, RemotePlayerService::class.java), serviceConnection, Service.BIND_AUTO_CREATE)
 
         // Handle window insets
         setStableLayoutFlags()
         ViewCompat.setOnApplyWindowInsetsListener(rootView) { v, insets ->
-            v.updatePadding(top = insets.systemWindowInsetTop, bottom = insets.systemWindowInsetBottom)
+            val layoutParams = webView.layoutParams as CoordinatorLayout.LayoutParams
+            layoutParams.updateMargins(top = insets.systemWindowInsetTop, bottom = insets.systemWindowInsetBottom)
```

Comment:
Does setting the padding not work for CoordinatorLayout? :flushed:

Reply:
It does, but it will make the snackbar appear in a wrong position. So i'm now using margins on the webview to prevent that :)